### PR TITLE
Documents Update & Agent v6.5.1 Release Notes

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? 
    - [Mobile-7449] A new doc to clarify a known issue
    - [Mobile-7472] Update an existing doc
    - [Mobile-7473] Agent v6.5.1 release notes

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - New Relic Mobile Android
+  - Troubleshoot
+metaDescription: 'New Relic Android Agent cannot coexist with firebase PerfMon SDK'
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile/troubleshoot/firebase-incompatibility-android
+---
+
+## Problem
+
+Conflicts, crashes and errors will show when include both New Relic plugin and Firebase PerfMon SDK within an app.
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx:15]: Typo

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Solution
+
+New Relic Android Agent and FireBase PerfMon cannot coexist and be enabled at the same time, it will create an infinite loop in the instrumented code. We would recommend to disable either one to avoid this issue.
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx:19]: This is not completely true: for the most part Firebase and our agent co-exist just fine. Keep in mind that FB is made up of many packages and I've found it is on the the Performance SDK we have trouble with, and with that only certain features that involve class rewriting. The goal of the story was to isolate and call-out what we can't work with. However, it's probably better at the moment to simply say, "_use one or the other, not both_".

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

**Comment by ywang-nr** [comment]
@dmurray for support team to review

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+subject: Android agent
+releaseDate: '2022-04-13'
+version: 6.5.1
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.5.1.zip'
+---
+
+## Fixed in this release
+
+* Resolved a memory leak that occurred in long-lived apps.
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx:10]: Please add a reference to where that leak occurred, and what was leaked

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx (hunk 3 above)
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx]: Provided more details

**Comment by ywang-nr** [comment]
This PR is ready for review, please let me know before your merge. Thank you.

**Comment by ywang-nr** [comment]
** DO NOT MERGE ** Thank you

**Pull request closed without merging**